### PR TITLE
Added more proxmox step templates

### PR DESCRIPTION
**Pull request by domrichardson**
Added more Proxmox steps to 
* Stop LXC
* Start LXC
* Destroy LXC

Fixed an issue with the Deploy LXC step template running in the background.

### CODE REVIEW

File: step-templates/proxmox-deploy-lxc.json [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
...etworkIndex++;\n}\n\nWrite-Host \"New LXC Summary:\"\n\n$LXCData | Convertto-json -depth 10\n\n$LXCCreateAsyncTask = (Invoke-RestMethod -Method POST -uri \"$($BaseURL)/nodes/$($ProxmoxNode)/lxc\" -Headers $header -Body $LXCData)\n\n\n$count = 1;\n$maxCount = 10;\n\nDO\n{\n \n $TaskID = $LXCCreateAsyncTask.Data;\n    Write-Host \"Checking if LXC has finished creating..\"\n    $LXCCreateAsyncTaskStatus = (Invoke-RestMethod -Method GET -uri \"$($BaseURL)/nodes/$($ProxmoxNode)/tasks/$($TaskID)/status\" -Headers $header).data\n    \n    if($LXCCreateAsyncTaskStatus.status -eq \"stopped\"){\n    \tif($LXCCreateAsyncTaskStatus.exitstatus -ne \"OK\"){\n        \tWrite-Error \"LXC create task finished with error: $($LXCCreateAsyncTaskStatus.exitstatus)\"\n        }else{\n        \tWrite-Host \"LXC create task has successfully completed!\"\n        }\n        \n        break;\n    }\n    \n\tWrite-Host \"LXC create task has not finished yet, retring in 5 seconds..\"\n    Write-Host \"Task Status: $($LXCCreateAsyncTaskStatus.status)\"\n    sleep 5\n    \n    If($count -gt $maxCount) {\n      Write-Host \"Task Timed out!\"\n      break;\n    }\n    $count++\n\n} While ($count -le $maxCount)\n"
```
Comment (harrisonmeister) [inline step-templates/proxmox-deploy-lxc.json:12]: If the task times out, should you change this:

```ps
Write-Host "Task Timed out!"
```
to a warning e.g.
```ps
Write-Warning "Task Timed out!"
```

?

File: step-templates/proxmox-deploy-lxc.json [hunk 2]
```diff
@@ -171,16 +181,6 @@
...
-			"Name": "Proxmox.LXC.Password",
-			"Label": "LXC Root Password",
-			"HelpText": "This will be the root password once the LXC container has been created",
-			"DefaultValue": "",
-			"DisplaySettings": {
-				"Octopus.ControlType": "Sensitive"
-			}
 		}
 	],
 	"StepPackageId": "Octopus.Script",
```
Comment (harrisonmeister) [inline step-templates/proxmox-deploy-lxc.json:186]: The `ExportedAt` should ideally be updated to indicate a change has also occurred :)

File: step-templates/proxmox-destroy-lxc.json [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
...Force -eq $True){\n  \t$LXC_Force = 1\n  }\n  \n} catch {}\n\n$LXCDestroyAsyncTask = (Invoke-RestMethod -Method DELETE -uri \"$($BaseURL)/nodes/$($ProxmoxNode)/lxc/$($LXC_VMID)?force=$($LXC_Force)\" -Headers $header)\n\n$count = 1;\n$maxCount = 10;\n\n$TaskID = $LXCDestroyAsyncTask.Data;\n\nDO\n{\n    Write-Host \"Checking if LXC has finished Destroying..\"\n    $LXCSDestroyAsyncTaskStatus = (Invoke-RestMethod -Method GET -uri \"$($BaseURL)/nodes/$($ProxmoxNode)/tasks/$($TaskID)/status\" -Headers $header).data\n    \n    if($LXCSDestroyAsyncTaskStatus.status -eq \"stopped\"){\n    \tif($LXCSDestroyAsyncTaskStatus.exitstatus -ne \"OK\"){\n        \tWrite-Error \"LXC destroy task finished with error: $($LXCSDestroyAsyncTaskStatus.exitstatus)\"\n        }else{\n        \tWrite-Host \"LXC destroy task has successfully completed!\"\n        }\n        \n        break;\n    }\n    \n\tWrite-Host \"LXC destroy task has not finished yet, retring in 5 seconds..\"\n    Write-Host \"Task Status: $($LXCSDestroyAsyncTaskStatus.status)\"\n    sleep 5\n    \n    If($count -gt $maxCount) {\n      Write-Host \"Task Timed out!\"\n      break;\n    }\n    $count++\n\n} While ($count -le $maxCount)\n"
```
Comment (harrisonmeister) [inline step-templates/proxmox-destroy-lxc.json:12]: Same thing here about the task time out

```ps
Write-Warning "Task Timed out!"
```

File: step-templates/proxmox-stop-lxc.json [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        "HelpText": "The LXC VMID you want to shutdown",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "94de0cbc-b722-4788-858f-fb2b32b57fd8",
+        "Name": "Proxmox.LXC.Stop",
+        "Label": "Force Stop LXC",
```
Comment (harrisonmeister) [inline step-templates/proxmox-stop-lxc.json:88]: Is this label accurate? The code implies if this is false, then it will shutdown, not force a stop?

**Review by harrisonmeister — CHANGES_REQUESTED** [review]
nothing major, a few minor changes

File: step-templates/proxmox-stop-lxc.json (hunk 4 above)
Comment (domrichardson) [reply step-templates/proxmox-stop-lxc.json:88]: If this parameter is enabled it will Force stop the LXC Container instead of gracefully shutting down

Comment (domrichardson) [reply step-templates/proxmox-stop-lxc.json:88]: Updated help text to better describe the parameter.

**Review by harrisonmeister — APPROVED** [review]

**Pull request merged**